zarr/_storage/store.py

```diff
@@ -129,6 +131,32 @@ def _ensure_store(store: Any):
             f"wrap it in Zarr.storage.KVStore. Got {store}"
         )
 
+    def getitems(self, keys: Sequence[str], meta_array: NDArrayLike) -> Mapping[str, Any]:
+        """Retrieve data from multiple keys.
+
+        Parameters
+        ----------
+        keys : Iterable[str]
+            The keys to retrieve
+        meta_array : array-like
+            An array instance to use for determining the output type. For now, this is
+            only a hint and can be ignore by the implementation, in which case the type
```

Comment:
"for now...only a hint" -- what's the plan after this PR? Will there be compatibility issues?

Reply:
In the future, we might require stores to return an array of type `meta_array`, which would introduce compatibility issues. Should I can remove the `For now`?

Reply:
Not sure I'm the best to say, but :+1: for thinking through those future compatibility issues.